### DISCUSSION TITLE
An option to copy to main Google calendar instead of a separate calendar?

### DISCUSSION THREAD

**Discussion by iman-prs** [derekantrican/GAS-ICS-Sync#395] [Q&A] [answered]
The idea is to post the events in the main Google calendar (and not to list it as a separate calendar in the calendar list) so that you can publish the calendar directly.

My specific problem is that I want to connect my Google calendar to a booking service which will read my availability status (and can create/modify/delete its own events). I want my calendar to show "busy" for the times I am busy on my work calendar (in addition to my personal calendar). However, I cannot grant editing access to my work calendar, so it can't be linked to this service directly. So I was trying to use this script to add my work calendar in "my calendars" but it turns out that only the main Google calendar is what is being shared (regardless of the sharing option of this one).

**Comment by jonas0b1011001** [comment, marked as answer]
Use your googlemail address as targetCalendar to sync into the main calendar.

**Reply by iman-prs** [reply]
Thank you!
I had tried using the name of my main calendar (and the link to it) as the targetCalendar but it kept creating a new one. The email address worked.
